src/Microsoft.Health.Fhir.Liquid.Converter/Hl7v2/Hl7v2DataParser.cs

```diff
@@ -81,8 +81,14 @@ private List<Hl7v2Field> ParseFields(string dataString, Hl7v2EncodingCharacters
                 {
                     if (!string.IsNullOrEmpty(fieldValues[f]))
                     {
-                        var components = ParseComponents(fieldValues[f], encodingCharacters);
-                        var field = new Hl7v2Field(fieldValues[f], components);
+                        /**
+                         * Take care of repeatable fields firstly, then render the whole field.
+                         * 1. If the config doesn't point out the field actually contains $RepetitionSeparator as repeatable,
```

Comment:
What does `config` refer to here?

Reply:
The `encodingCharacters`.

Reply:
Got it. Let's update the comment here to be more specific.